system_tests/logging_.py

```diff
@@ -175,3 +177,25 @@ def test_create_sink_bigquery_dataset(self):
         sink.create()
         self.to_delete.append(sink)
         self.assertTrue(sink.exists())
+
+    def test_create_sink_pubsub_topic(self):
+        from gcloud import pubsub
+        TOPIC_URI = 'pubsub.googleapis.com/projects/%s/topic/%s' % (
+            Config.CLIENT.project, TOPIC_NAME)
```

Comment:
ATM, we have punted on #1591, due to your legitimate concerns about inter-package coupling.